tests/tracing/test_provider.py

```diff
@@ -126,6 +152,13 @@ def foo():
     mlflow.tracing.enable()
     assert _is_enabled()
 
+    # _is_enabled() should not raise an exception
+    with mock.patch(
+        "mlflow.tracing.provider.trace.get_tracer", side_effect=ValueError("error")
+    ) as get_tracer_mock:
+        assert not _is_enabled()
```

Comment:
```suggestion
        assert _is_enabled() is False
```

to ensure `False` is returned?